README.md

```diff
@@ -34,7 +34,7 @@
 - **Controller** receives data from the monitor and node exporter, and controls the whole workflow of our system.
```

Comment:
I think the notebook section is a part of tutorial section, you can append the notebook links below that.